pkgs/top-level/all-packages.nix

```diff
@@ -26838,6 +26838,8 @@ with pkgs;
 
   system-sendmail = lowPrio (callPackage ../servers/mail/system-sendmail { });
 
+  unFTP = callPackage ../servers/unftp { };
+
```

Comment:
The package should be placed under the `pkgs/by-name/so/some-package/package.nix` hierarchy. Which would make this unnecessary.

```suggestion
```